### DISCUSSION TITLE
I dont see the tidal prompt in my system log

### DISCUSSION THREAD

**Discussion by yamimizu832** [RandomNinjaAtk/docker-lidarr-extended#220] [Q&A] [answered]
I setup a docker container for lidarr-extended and I do not see the prompt in the system log for tidal dl to go to the URL for validation. Below is my docker compose: 

version: "2.1"
services:
  lidarr-extended:
    image: randomninjaatk/lidarr-extended:latest
    container_name: lidarr-extended
    volumes:
      - /home/ravanafr/Lidarr-test/config:/config
      - /home/ravanafr/Lidarr-test/downloads:/downloads-lidarr-extended
      - /home/ravanafr/Lidarr-test/music:/music
      - /home/ravanafr/Lidarr-test/music videos:/music-videos
    environment:
      - TZ=America/New_York
      - PUID=0
      - PGID=0
      - enableAudioScript=true
      - enableVideoScript=true
      - scriptInterval=15m
      - configureLidarrWithOptimalSettings=true
      - searchSort=date
      - audioFormat=native
      - audioBitrate=lossless
      - requireQuality=true
      - enableReplaygainTags=true
      - audioLyricType=both
      - dlClientSource=Tidal
      - arlToken=Token_Goes_Here
      - tidalCountryCode=AR
      - addDeezerTopArtists=true
      - addDeezerTopAlbumArtists=true
      - addDeezerTopTrackArtists=true
      - topLimit=10
      - addRelatedArtists=false
      - numberOfRelatedArtistsToAddPerArtist=5
      - lidarrSearchForMissing=true
      - addFeaturedVideoArtists=false
      - plexUrl=http://x.x.x.x:32400
      - plexToken=Token_Goes_Here
      - youtubeSubtitleLanguage=en
      - enableQueueCleaner=true
      - matchDistance=5
      - enableBeetsTagging=true
      - beetsMatchPercentage=90
      - retryNotFound=90
    ports:
      - 6686:6686
    restart: unless-stopped
      
  qbittorrent:	
    image: lscr.io/linuxserver/qbittorrent
    container_name: qbittorrent
    ports:
     - 8080:8080
    environment:
     - PUID=0
     - PGID=0
     - TZ=America/New_York
    volumes:
     - /home/ravanafr/lidarqbit/config :/config
     - /home/ravanafr/lidarqbit/downloads :/downloads
    restart: unless-stopped

When I am looking at the system log I don't see the authentication prompt. Below is what I normally see:

2023-04-07 19:40:34.6|Info|Bootstrap|Starting Lidarr - /app/lidarr/bin/Lidarr.dll - Version 1.1.3.2982
2023-04-07 19:40:34.8|Info|AppFolderInfo|Data directory is being overridden to [/config]
2023-04-07 19:40:35.4|Info|AppFolderInfo|Data directory is being overridden to [/config]
2023-04-07 19:40:35.6|Info|MigrationController|*** Migrating data source=/config/lidarr.db;cache size=-10000;datetimekind=Utc;journal mode=Wal;pooling=True;version=3 ***
2023-04-07 19:40:35.9|Info|MigrationController|*** Migrating data source=/config/logs.db;cache size=-10000;datetimekind=Utc;journal mode=Wal;pooling=True;version=3 ***
2023-04-07 19:40:36.9|Info|Lidarr.Http.Authentication.ApiKeyAuthenticationHandler|AuthenticationScheme: SignalR was challenged.

Any help will be appreciated. I am very new to docker so I'm not sure if I have done something wrong.

**Comment by RandomNinjaAtk** [comment]
Your likely looking in the wrong place.

https://github.com/RandomNinjaAtk/docker-lidarr-extended/discussions/17#discussioncomment-3224672


You'll want to looking in the Audio log file, or just watch the container logs.

**Reply by yamimizu832** [reply]
Hello and thank you for responding. I saw that question answer (17) before I created my own. I don't see an extended_script_run_*.txt file in the config folder.  I've checked in the config/extended/ logs folder and the config/extended/scripts folder. I don't see that file being created. I also downloaded portainer and was using that to monitor the log on my docker container and I don't see anything as I sit and watch the log showing a prompt. I also looked in the config/log section but that also just shows what is available from the console log in lidarr since they are the same thing. If  I am not supposed to be looking in these locations where should I be looking? I a

**Reply by yamimizu832** [reply]
I am unsure of what the problem was. I went ahead and just redid the entire container. After I did this I was able to see the prompt in the live log of the container via portainer. Unsure why the first container did not work since I copied the exact same docker compose to the new container. Thank you for attempting to assist.

**Comment by RandomNinjaAtk** [comment, marked as answer]
Just for future reference, logs are stored in the `/config/logs` folder. Log files are named after the scripts, so for audio (`Audio.txt`) and video (`Video.txt`).

The logs are viewable via the Lidarr web interface, navigate to: Lidarr -> System -> Log Files.

Additionally, you should also see the script logging output in the overall containers logging output.
